### PR TITLE
Update iot-device-update info

### PR DESCRIPTION
**Pull request by Saisang**

### CODE REVIEW

**Comment by Saisang** [comment]
@czubair Do you know why the link broken? The file is here https://learn.microsoft.com/en-us/javascript/api/overview/azure/iot-device-update-rest-readme

File: _data/releases/latest/js-packages.csv [hunk 1]
```diff
@@ -425,7 +425,7 @@
 "@azure-rest/arm-containerservice","","1.0.0-beta.1","@azure-rest/arm-containerservice","","NA","NA","NA","","false","","","","","","","","",""
 "@azure-rest/arm-servicefabric","","1.0.0-beta.1","@azure-rest/arm-servicefabric","","NA","NA","NA","","false","","","","","","","","",""
 "@azure-rest/core-client-lro","","1.0.0-beta.1","@azure-rest/core-client-lro","","NA","NA","NA","","false","","","","","","","","",""
-"@azure-rest/iot-device-update","1.0.0","","@azure-rest/iot-device-update","","NA","NA","NA","","false","","","","","","","","",""
+"@azure-rest/iot-device-update","1.0.0","","IoT Device Update","IoT Device Update","https://github.com/Azure/azure-sdk-for-js/tree/@azure-rest/iot-device-update_item.Version/sdk/deviceupdate/iot-device-update-rest","","","client","true","","","","","","","","",""
```
[the remark this answers is not in the document]
Comment (weshaggard) [reply _data/releases/latest/js-packages.csv:428]: @czubair the azure-rest packages are special in that they don't follow the normal paths in the repo. What is there is very close but we don't want that `@`.

[the remark this answers is not in the document]
Comment (weshaggard) [reply _data/releases/latest/js-packages.csv:428]:
```suggestion
"@azure-rest/iot-device-update","1.0.0","","IoT Device Update","IoT Device Update","https://github.com/Azure/azure-sdk-for-js/tree/@azure-rest/iot-device-update_item.Version/sdk/deviceupdate/iot-device-update-rest","NA","NA","client","true","","","","","","","","",""
```

**Review by weshaggard — APPROVED** [review]

File: _data/releases/latest/js-packages.csv (hunk 1 above)
[the remark this answers is not in the document]
Comment (czubair) [reply _data/releases/latest/js-packages.csv]: Okay, thanks @weshaggard

**Pull request merged**